lib/web_ui/test/ui/platform_view_test.dart

```diff
@@ -120,10 +120,14 @@ Future<void> testMain() async {
     sb.pushOffset(0, 0);
     sb.addPicture(const ui.Offset(100, 100), recorder.endRecording());
 
+    // Nest offsets both before and after the transform to make sure that they
+    // are applied properly.
+    sb.pushOffset(50, 50);
     sb.pushTransform(Matrix4.rotationZ(0.1).toFloat64());
+    sb.pushOffset(25, 25);
```

Comment:
Maybe make a separate copy of the test? The previous scenario with a transform that's not sandwiched between two offsets seems like an interesting case to test too.

Reply:
Good idea.